register-service-provider.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+. .env
+
+IMAGE=$(grep "raidennetwork/raiden-services:" docker-compose.yml|cut -d ':' -f2-|xargs)
```

Comment:
Suggestion: Use bash magic to reduce need for external tools:
```bash
IMAGE=${$(grep "raidennetwork/raiden-services:" docker-compose.yml)/*image:*/}
```
(From: https://stackoverflow.com/a/13210909/107946)

Reply:
wow, nice!

Reply:
meh...while it works on commandline, I can't seem to get the quoting right inside the script :(